### PR TITLE
Admin has ability to modify Users role (REST API)

### LINKED ISSUE #2625: Users API - admin need ability to modify user roles
When Users REST API is used to modify user's data, admin needs to be able to modify user roles, too.

### PR DESCRIPTION
**Pull request by marla-singer**
Closes #2625

## Developer checklist
This checklist is to be completed by the PR developer:

- [ ] Alternative solutions were compared/discussed before writing code
    - [ ] trade-offs with this solution are considered acceptable
- [ ] Code in this PR adheres to the [project styleguide](CONTRIBUTING.md)
- [ ] This pull request does not decrease project test coverage
- [ ] If the code changes existing database collection(s), migration has been written
- [ ] If UI texts are added or changed, all texts are internationalized

## Reviewer checklist
Reviewed by: @matleppa 

This list is to be completed by the pull request reviewer:
- [ ] Code works as described/expected
- [ ] Code seems to be error free
    - [ ] no browser console errors visible
    - [ ] no server console errors visible
    - [ ] passes CI build
- [ ] Code is written in a way that promotes maintainability
    - [ ] easy to understand
    - [ ] well organized
    - [ ] follows project coding standards and conventions
    - [ ] well documented

### CODE REVIEW

File: apinf_packages/rest_apis/server/management.js [hunk 1]
```diff
@@ -51,6 +51,14 @@ ManagementV1.swagger = {
     users: 'Users',
   },
   params: {
+    addRole: {
+      name: 'role',
+      in: 'body',
+      description: 'A role value to add',
+      required: true,
```
Comment (matleppa) [inline apinf_packages/rest_apis/server/management.js]: When parameter type is body, it needs schema definition included. 
Same way as e.g. in login parameter below.

Without schema definition swagger compiler fails, when document is added into API card.

Remake addRole parameter to contain schema.

Comment (marla-singer) [reply apinf_packages/rest_apis/server/management.js]: Ok

Comment (marla-singer) [reply apinf_packages/rest_apis/server/management.js]: Done

**Comment by matleppa** [comment]
Hi @marla-singer 
There are still needed some corrections in definitions.

### params:
  addRole

**Remove fields:**
      required: true,
      type: 'string',
      enum: ['admin', 'manager'],

### Definitions
  addRole

**Add into role**
          enum: ['admin', 'manager'],

**Comment by marla-singer** [comment]
@matleppa Sorry, I don't understand. Can you provide the total correct piece of code that should be and provide a place that should place?

**Comment by matleppa** [comment]
Following changes are needed in order to get the parser accept the json file.

 ```
 params: {
    addRole: {
      name: 'role',
      in: 'body',
      description: 'A role value to add',
      schema: {
        $ref: '#/definitions/addRole',
      },
    },



  definitions: {
    addRole: {
      required: ['role'],
      properties: {
        role: {
          type: 'string',
          enum: ['admin', 'manager'],
          example: 'manager',
        },
      },
    },

```

**Comment by marla-singer** [comment]
@matleppa Fixed

**Pull request merged**